src/fiatconnect/fiatAccountSchemas/types.ts

```diff
@@ -4,6 +4,13 @@ import { KeyboardType } from 'react-native'
 export interface FormFieldParam {
   name: string
   label: string
+  infoDialog?: {
+    title: string
+    actionText: string
+    body: string
+    testID?: string
```

Comment:
Could we just automatically add testID using name instead of making this a form field param. E.g., we can generate testID like `${name}/dialog`

Reply:
Added with the format `dialog-${name}` to match other test ids in the file.